scripts/publish.sh

```diff
@@ -5,8 +5,8 @@ set -e
 set -o pipefail
 
 if [[ -z $NPM_TOKEN ]]; then
```

Comment:
If we change the name of the environment variable should we also change this reference?

Reply:
Oops, good catch!

Reply:
Actually.... we shouldn't need to set this in config.Unsure why that step was even here.